indico/util/roles.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+import csv
+
+from flask import flash
+
+from indico.core.errors import UserValueError
+from indico.modules.events.roles.forms import ImportMembersCSVForm
+from indico.modules.users import User
+from indico.util.i18n import _, ngettext
+from indico.util.string import to_unicode, validate_email
+from indico.web.flask.templating import get_template_module
+from indico.web.util import jsonify_data, jsonify_template
+
+
+class ImportRoleMembersMixin():
+
```

Comment:
don't create old-style classes - always inherit from `object` if there's no other base class. also, no empty line after the class header (it's only needed after docstrings)